src/neptune/new/utils.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+__all__ = ["stringify_unsupported"]
+
+from typing import (
+    List,
+    Mapping,
+    Union,
+)
+
+from neptune.new.internal.utils.stringify_value import StringifyValue
+
+
+def stringify_unsupported(value) -> Union[StringifyValue, Mapping, List]:
+    if isinstance(value, dict):
```

Comment:
Maybe we should use `is_dict_like` or even take `argaprse.Namespace` and `neptune.Namespace`.

I don't know, for me it's a problematic topi I've addressed earlier.

Reply:
I know, I know. I just wonder about covering the cases mentioned in the spec. Not sure what I have to cover. The `argparse.Namespace` works correctly:
```python
run['a/with_warning'] = stringify_unsupported(argparse.Namespace(test="test", test2=5.3))
run['a/with_warning'] = argparse.Namespace(test="test", test2=5.3)
```
and doesn't generates any warnings in both usages (as it's a supported type).

Reply:
What about `argparse.Namespace(test=SomeClass(), test2=5.3)`?